lib/wickedbase.pm

```diff
@@ -46,6 +46,9 @@ sub get_ip {
     elsif ($args{type} eq 'tunl1') {
         return $args{is_wicked_ref} ? '3.3.3.10' : '3.3.3.11';
     }
+    elsif ($args{type} eq 'tun1') {
```

Comment:
I haven't seen any getip calls in code, so why we need this?

Reply:
it is done by get_test_result()

Reply:
in this case, we need to change ifcfg files too. The main idea to keep setting in one place, so let's follow the same pattern here too - ips are only inside get_ip and you have logic which replaces anchor words in config files with data gotten from get_ip

Reply:
Done, check last patch. I created a function in wickedbase that is used by ref and sut. I can't reuse setup_tunnel because the local and remote IPs  are not the 'host' ones.